packages/web/src/components/pages/Tests/TestDetails/TestSettings/SettingsVariables/Arguments.tsx

```diff
@@ -28,6 +28,19 @@ import {prettifyArguments} from '@utils/prettifyArguments';
 
 import {ArgumentsWrapper} from './Arguments.styled';
 
+const formatArgsArray = (args: string[]) => {
+  return args.map(arg => {
+    const trimmedArg = arg.replace(/\s+|(?<=[^'])['"](?=[^'])/g, match => {
+      if (match.trim() === '') {
+        return ' '; // Replaces multiple spaces with a single space
```

Comment:
It should not replace multiple spaces in a context of single argument.

It would replace:
```yaml
# abc "   def  xyz   " "abc"
command:
  - abc
args:
  - "   def  xyz   "
  - "abc"
```

With:
```yaml
# abc " def xyz " "abc"
command:
  - abc
args:
  - " def xyz "
  - "abc"
```